_includes/docs/latest/azkaban-webserver.html

```diff
@@ -55,6 +55,60 @@ <h2 id="azkaban-webserver">Azkaban Web Server Configurations</h2>
 	</tbody>
 </table>
 
+<h3>Multiple Executor Mode Parameters</h3>
+
+<table class="table table-striped table-bordered table-condensed">
+	<thead>
+		<tr>
+			<th>Parameter</th>
+			<th>Description</th>
+			<th>Default</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td>azkaban.use.multiple.executors</td>
+			<td>Should azkaban run in multi-executor mode. Required for multiple executor mode.</td>
+			<td>false</td>
+		</tr>
+		<tr>
+			<td>azkaban.executorselector.filters</td>
+			<td>Hard filters to be used while dispatching. To be choosen from StaticRemaining, FlowSize, MinimumFreeMemory and CpuStatus.</td>
+			<td> </td>
+		</tr>
+		<tr>
+			<td>azkaban.executorselector.comparator.{ComparatorName}</td>
```

Comment:
Multiple will have separate property. As discussed offline,  I have added an example and made specification more clearer by specifying that it is an integer weight and what can be the value for Memory {ComparatorName}.